dev-vcs/git-publish/git-publish-1.7.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+inherit python-single-r1
+
+DESCRIPTION="Tool for preparing and storing patch revisions as git tag"
+HOMEPAGE="https://github.com/stefanha/git-publish"
+SRC_URI="https://github.com/stefanha/git-publish/archive/v${PV}.tar.gz -> $P.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="man"
+
+DEPEND=""
+RDEPEND="${PYTHON_DEPS}
+	>=dev-lang/python-3.8:*
```

Comment:
```suggestion
```

The eclass [handles](https://dev.gentoo.org/~mgorny/python-guide/single.html#basic-use-for-unconditional-python) this for us - -we don't want to interfere with it.

Reply:
.. and because we're not adding anything custom, we can actually not define RDEPEND or DEPEND at all as it does it for us (it sets it to ${PYTHON_DEPS} automatically)

Reply:
Maybe you confused this situation with situations where `distutils-r1.eclass` is used.
`distutils-r1.eclass` sets e.g. `BDEPEND` or `RDEPEND`, but `python-single-r1.eclass` does not.
If this package has only pure-Python modules (no extension modules written in C/C++/Fortran/...), then the following are still needed:
```
BDEPEND="${PYTHON_DEPS}"
RDEPEND="${PYTHON_DEPS}
	dev-vcs/git
"
```

Reply:
ah, yes